clients/pricing-client/src/openapi.d.ts

```diff
@@ -2038,6 +2031,13 @@ declare namespace Components {
         }
         export interface PriceGetAg {
             category: string;
+            markup_pricing_model?: "per_unit" | "tiered_volume" | "tiered_flatfee";
```

Comment:
Do we have already a component schema for Pricing Mode? if so maybe we can use here (knowing that the cumulative will not be possible yet). What you think?

Reply:
We don't have, I can create it. But it's better if we have one for the pricing models and another for markup pricing model.

The `cumulative` is not the only diff. On pricing we also have the new `get-ag` model